Google_CLoud_Functions/main.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+home = "https://svc-milanvelle-dot-rbfa-workshop-sandboxes.ew.r.appspot.com"
+één = "https://svc-milanvelle-dot-rbfa-workshop-sandboxes.ew.r.appspot.com/bq_insert_timestamp"
```

Comment:
- do not use special character in names
- of you start writing in English keep writing in English

Reply:
you define the variable "home" why don't you reuse it? For example, one = home + "/bq_insert_timestamp" for example?